style.css

```diff
@@ -0,0 +1,1821 @@
+@import url("https://fonts.googleapis.com/css2 family=Poppins:wght@300&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: start;
+  align-content: stretch;
+  box-sizing: border-box;
+  width: 375px;
+  height: 6000px;
```

Comment:
- [ ] [OPTIONAL] Kindly set your height property to **auto** instead and add a **padding-bottom** to have vertical spacing at the bottom 😁 It would render your design more flexible ❤️ 

Your current design 👇🏽 👇🏽 

![Screenshot from 2023-06-08 07-10-42](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/10439283/fa8fe271-af8f-4d2f-a495-5af057c6041f)